api/types/database.go

```diff
@@ -922,6 +922,11 @@ func (d *DatabaseV3) CheckAndSetDefaults() error {
 		}
 	}
 
+	const defaultKRB5FilePath = "/etc/krb5.conf"
+	if d.Spec.AD.Domain != "" && d.Spec.AD.Krb5File == "" {
```

Comment:
Could you add a comment on why checking `d.Spec.AD.Domain !=""` here? (or move the old comment ?)